### DISCUSSION TITLE
Mayor personalización de espacios de tiempo

### DISCUSSION THREAD

**Discussion by ddi4z** [Open-Source-Uniandes/Mi-Horario-Uniandes#53] [Ideas]
Podría incluirse la posibilidad de editar el intervalo de tiempo de un espacio de tiempo. También, podrían añadirse nuevas características como la posibilidad de darle nombre a los espacios de tiempo.

**Comment by c4ts0up** [comment]
Listo, ya quedó planeado 👌🏽
